pkg/ratelimiter/ratelimiter.go

```diff
@@ -0,0 +1,81 @@
+package ratelimiter
```

Comment:
do we need the copyright boilerplate?

Reply:
Sorry I did not see it on other files so I forgot it.
I will add it.